tests/test_expval.py

```diff
@@ -178,18 +183,25 @@ def test_hermitian_expectation(self, n_wires, theta, phi, qubit_device, tol):
             obs = qml.Hermitian(U, wires=perm)
 
             def circuit():
-                qml.StatePrep(init_state, wires=range(n_qubits))
+                if device_name != "lightning.tensor":
+                    qml.StatePrep(init_state, wires=range(n_qubits))
                 qml.RY(theta, wires=[0])
                 qml.RY(phi, wires=[1])
                 qml.CNOT(wires=[0, 1])
                 return qml.expval(obs)
 
             circ = qml.QNode(circuit, dev)
             circ_def = qml.QNode(circuit, dev_def)
-            assert np.allclose(circ(), circ_def(), tol)
+            if device_name != "lightning.tensor":
+                assert np.allclose(circ(), circ_def(), tol)
+            else:
+                assert np.allclose(circ(), circ_def(), rtol=0.25)
 
 
-@pytest.mark.parametrize("diff_method", ("parameter-shift", "adjoint"))
+@pytest.mark.parametrize(
+    "diff_method",
+    ("parameter-shift", "adjoint" if device_name != "lightning.tensor" else "parameter-shift"),
```

Comment:
Shouldn't we just use `pytest.skip` if `lightning.tensor` instead of running the test twice?

Reply:
we don't want to skip all tests here since we need to test `Sprod` etc for `lightning.tensor`. However, we can skip `adjoint` here with `pytest.mark.skipif`